server/graphql/v2/mutation/UserMutations.js

```diff
@@ -0,0 +1,28 @@
+import { GraphQLNonNull } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date/dist';
```

Comment:
Any reason for the `/dist`? We're not doing this elsewhere.

Reply:
@znarf : Yes, you are right. The `dist` shouldn't be there. I've corrected that. 😄